tools/templates/Makefile-backend-common.in

```diff
@@ -118,7 +118,7 @@
 @stage_precomp(@bsm(P6QREGEX_@ucstage@)@:					@use_prereqs(@bpm(P6QREGEX_COMBINED_@ucstage@)@)@ @bsm(@ucstage@_DEPS)@ @bsm(HLL_@ucstage@)@ @bsm(QREGEX_@ucstage@)@ @bsm(QAST_@ucstage@)@ @bsm(QASTNODE_@ucstage@)@)@
 
 
-@nfp(@stage_dir@/nqp-config.nqp)@:
+@nfp(@stage_dir@/nqp-config.nqp)@: @bpm(SOURCES)@
```

Comment:
Previously this recipie was part of the following target and depended on `@bsm(NQP_@ucprev_stage@)@ @nfp(@stage_dir@/@bsm(QAST)@)@ @nfp(@stage_dir@/@bsm(P6QREGEX)@)@ @bpm(SOURCES)@`

Does `@bpm(SOURCES)@` already contain all `.nqp` files from `src/`?

Btw. now that I see this, I think we have always missed `MAST/Ops.nqp` and `MAST/Nodes.nqp` in `gen-version.pl`

Reply:
On moar it does with the exception of `P6QREGEX_SOURCES`. I'm not sure if that's missing on purpose.
On JVM it includes everything.
I didn't check on JS.